static-assets/components/cstudio-contextual-nav/wcm-site-dropdown-mods/wcm-root-folder.js

```diff
@@ -1291,19 +1329,19 @@
 	                			node.isLeaf = true;
 	                		}
 
-	                		Self.drawSubtree(treeData.item.children, args.node, args.pathToOpenTo, args.instance);
```

Comment:
@sumerjabri this is another reason to call up GitHub support. This part wasn't really changed in content but it's getting modified marks and as a reviewer is confusing.